src/main/java/com/codicesoftware/plugins/jenkins/tools/CmToolInstaller.java

```diff
@@ -58,7 +59,7 @@ public FilePath performInstallation(
         // Skip system tool check if it was previously installed
         // Admins can always disable the automatic installer to fall back to the path they specified for the tool
         if (!ignoreSystemTool && !installablePath.child(".installedFrom").exists()) {
-            FilePath existingHome = new FilePath(node.getChannel(), tool.getHome());
+            FilePath existingHome = new FilePath(node.getChannel(), Objects.toString(tool.getHome(), ""));
```

Comment:
As the intent is to prevent a null from `tool.getHome()`, let's use `hudson.Util.fixEmpty()` as other parts in the code.

```suggestion
            FilePath existingHome = new FilePath(node.getChannel(), Util.fixEmpty(tool.getHome()));
```